docs/release-notes/8.11.asciidoc

```diff
@@ -1,6 +1,18 @@
 [[release-notes-header-8.11.0]]
 == 8.11
 
+[discrete]
+[[release-notes-8.11.4]]
+=== 8.11.3
+
+[discrete]
+[[bug-fixes-8.11.4]]
+==== Bug fixes
+* Stops the {esql} tab from rendering when Timeline isn't open ({pull}173484[#173484]).
+* Fixes a bug that caused the **Add to Case** action to fail if you didn't add a comment before isolating and releasing a host ({pull}172912[#172912]).
+* Fixes the {esql} default query size and appends the query with a `keep` clause ({pull}171789[#171789]).
```

Comment:
As discussed with @PhilippeOberti today, this change will be overwritten (dev PR pending), so I might need to remove this PR.

cc: @michaelolo24

Reply:
Verified with @kqualters-elastic via slack that this can be removed.

```suggestion
```